### PR TITLE
Create Wumpus Stub

### PR DESCRIPTION
**Pull request by ILHS2000503**

### CODE REVIEW

**Review by MaximumMaxxx — DISMISSED** [review]
Can we switch state into an enum instead of an int

**Comment by ILHS2000503** [comment]
dont know what an enum is

File: Core/WumpusCore/Wumpus/Wumpus.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```
Comment (roguefirework) [inline Core/WumpusCore/Wumpus/Wumpus.cs:3]: Unneeded Import

File: Core/WumpusCore/Wumpus/Wumpus.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+using System.Security.Cryptography.X509Certificates;
```
Comment (roguefirework) [inline Core/WumpusCore/Wumpus/Wumpus.cs:4]: Uneeded

File: Core/WumpusCore/Wumpus/Wumpus.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+using System.Text;
```
Comment (roguefirework) [inline Core/WumpusCore/Wumpus/Wumpus.cs:5]: I think we dont need this

File: Core/WumpusCore/Wumpus/Wumpus.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+using System.Threading.Tasks;
```
Comment (roguefirework) [inline Core/WumpusCore/Wumpus/Wumpus.cs:6]: Dont see any threading code atm

**Review by roguefirework — CHANGES_REQUESTED** [review]
Can you quickly remove the unnesary imports

**Review by roguefirework — APPROVED** [review]
Looks good now

**Review by ilhs1041495 — APPROVED** [review]

**Review by 1118422ILHS — CHANGES_REQUESTED** [review]
Doc should be typed better

**Comment by ILHS2000503** [comment]
what doc?

**Comment by ILHS2000503** [comment]
Changed the docs to be a bit more detailed and specify that more detailed summaries will be added latter.

**Review by 1118422ILHS — APPROVED** [review]
Comments are better. "Better description coming soon" unnecessary but fine. Remember to use parameter and returns tags once necessary.

**Pull request merged**